packages/flutter/lib/src/material/user_accounts_drawer_header.dart

```diff
@@ -232,7 +232,7 @@ class _AccountDetailsLayout extends MultiChildLayoutDelegate {
     final String bottomLine = hasChild(accountEmail) ? accountEmail : (hasChild(accountName) ? accountName : null);
 
     if (bottomLine != null) {
-      final Size constraintSize = iconSize == null ? size : size - Offset(iconSize.width, 0.0);
+      final Size constraintSize = iconSize == null ? size : (size - Offset(iconSize.width, 0.0) as Size);
```

Comment:
Instead:

```dart
final Size constraintSize = iconSize == null ? size : size - Size(iconSize.width, 0.0);
```

Reply:
Your proposal also required a cast because `Size.-(OffsetBase other)` returns an `OffsetBase`

Reply:
Hm.  in that case I think we should just avoid the convenience operator here and do:

```
final Size constraintSize = iconSize == null ? size : Size(size.width - iconSize.width, size.height);
```

Reply:
Done